tests/fdb/test_fdb_flush.py

```diff
@@ -0,0 +1,362 @@
+
+FLUSH_TYPES = ["dynamic", "static", "interface", "mix"]
+
+FDB_SET_JSON_FILE = 'fdb_set_test.json'
+FDB_DEL_JSON_FILE = 'fdb_del_test.json'
+FDB_FILES_DIR = '/tmp/'
+DUT_WORKING_DIR = '/etc/sonic/'
+
+class TestFdbFlush:
+    """
+        TestFdbFlush Verifies FDb operation is smooth, No new core files are generated after FDb operation.
+        FDb operation:
+            - Swssconfig add static fdb
+            - Swssconfig remove static fdb
+            - Sonic-clear fdb all
+            - Send packets to create dynamic fdb
+            - interface shutdown
+            - interface startup
+
+        The test do FDb operation, and make sure no new core files are generated after the test.
+    """
+
+    DUMMY_MAC_PREFIX = "00:11:22:33:55"
+    FDB_INFO_FILE = "/tmp/fdb_info.txt"
+
+    def __getFdbTableCount(self, duthost, mac):
+        """
+            Gets number of FDB table entries containing mac entry.
+
+            Args:
+                duthost (AnsibleHost): Device Under Test (DUT)
+                mac (str): MAC value to search for in the FDB table
+
+            Returns:
+                (int) representing the number of the FDB entries containing th eMAC address
+        """
+        return int(duthost.shell("show mac | grep {0} | wc -l".format(mac))["stdout"])
+
+    def __loadSwssConfig(self, duthost):
+        """
+            Load SWSS configuration on DUT
+
+            Args:
+                duthost (AnsibleHost): Device Under Test (DUT)
+
+            Raises:
+                asserts if the load SWSS config failed
+
+            Returns:
+                None
+        """
+        duthost.shell(argv=[
+            "docker",
+            "exec",
+            "swss",
+            "bash",
+            "-c",
+            "swssconfig /etc/swss/config.d/switch.json"
+        ])
+
+    def __deleteTmpSwitchConfig(self, duthost):
+        """
+            Delete temporary switch.json cofiguration files
+
+            Args:
+                duthost (AnsibleHost): Device Under Test (DUT)
+
+            Returns:
+                None
+        """
+        result = duthost.find(path=["/tmp"], patterns=["switch.json*"])
+        for file in result["files"]:
+            duthost.file(path=file["path"], state="absent")
+
+    def __runPtfTest(self, ptfhost, testCase='', testParams={}):
+        """
+            Runs FDB MAC Expire test case on PTF host
+
+            Args:
+                ptfhost (AnsibleHost): Packet Test Framework (PTF)
+                testCase (str): FDB tests test case name
+                testParams (dict): Map of test params required by testCase
+
+            Returns:
+                None
+
+            Raises:
+                RunAnsibleModuleFail if ptf test fails
+        """
+        logger.info("Running PTF test case '{0}' on '{1}'".format(testCase, ptfhost.hostname))
+        ptf_runner(
+            ptfhost,
+            "ptftests",
+            testCase,
+            platform_dir="ptftests",
```

Comment:
@lipxu should platform_dir be "ptftests/py3" instead of "ptftests"?

Reply:
here we put the True to is_python3, I think the code would add "/py3". So maybe no need to use "ptftests/py3" instead of "ptftests" here.

    if is_python3:
        path_exists = host.stat(path="/root/env-python3/bin/ptf")
        if path_exists["stat"]["exists"]:
            cmd = "/root/env-python3/bin/ptf --test-dir {} {}".format(testdir+'/py3', testname)